internal/pkg/deploy/deploy.go

```diff
@@ -18,22 +24,138 @@ type CreateEnvironmentInput struct {
 	ToolsAccountPrincipalARN string // The Principal ARN of the tools account.
 }
 
+const (
+	GithubProviderName = "GitHub"
+	GithubSecretIdKeyName = "githubOAuthSecretId"
```

Comment:
> Nit: Are you referring to the Personal Access Token here? In the [github console](https://github.com/settings/tokens) they refer to it as such, which is what i've been doing in the UX so not sure if that will be confusing...

Yea Cloudformation calls it OAuthToken. Will rename because